packages/core/test/oracle/Voting.js

```diff
@@ -1502,12 +1502,17 @@ contract("Voting", function(accounts) {
 
     await voting.snapshotCurrentRound(signature);
 
-    assert(await didContractThrow(voting.revealVote(identifier, time, getRandomSignedInt(), getRandomUnsignedInt())));
-    await voting.revealVote(identifier, time, retrievedVote.price, retrievedVote.salt);
+    assert(await didContractThrow(voting.revealVote(identifier, time, getRandomSignedInt(), getRandomSignedInt())));
+    await voting.revealVote(
+      identifier,
+      time,
+      web3.utils.toBN(retrievedVote.price),
```

Comment:
similar issue as in `SignedFIxedPoint.js` where I have to cast a negative `int` to a BN before passing it as input into a function call